### PR TITLE
Adding stacks.js

### PR DESCRIPTION
**Pull request by scolear**

### CODE REVIEW

File: attestor/observer/package.json [hunk 1]
```diff
@@ -23,12 +26,20 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@stacks/blockchain-api-client": "^7.0.1",
```
Comment (sosaucily) [inline attestor/observer/package.json:29]: do you want to bump this stuff to the newest? If it works, might be a good time to do it, as we haven't done thorough testing yet anyway

File: wallet-blockchain-interface/src/chains/stacks/helper-functions.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    return hexToBytesMS(hex.substring(0, 2) === '0x' ? hex.substring(2) : hex);
+}
+
+/**
+ * Converts a lite signature to the format expected by Stacks. It merely
+ * subtracts 27 from the recovery byte and returns it as a Uint8Array.
+ * @param liteSignature
+ * @returns Uint8Array
+ */
+export function liteSignatureToStacksSignature(liteSignature: Uint8Array | string) {
```
Comment (sosaucily) [inline wallet-blockchain-interface/src/chains/stacks/helper-functions.ts:56]: ooo la la what is this for?

File: attestor/observer/src/chains/stacks/get-observer.ts [hunk 3]
```diff
@@ -1,6 +1,46 @@
...
+  const networkConfig: NetworkConfig = getConfig(config);
+
+  switch (config.version) {
+    case '1':
+      dlcManager = new DlcManagerV1(networkConfig.socket, networkConfig.deploymentInfo);
+      await dlcManager.init();
+      break;
+  }
+
+  return {
```
Comment (sosaucily) [inline attestor/observer/src/chains/stacks/get-observer.ts:19]: Would be nice to not need to worry about all this microblock stuff one day

File: attestor/observer/package.json (hunk 1 above)
Comment (scolear) [reply attestor/observer/package.json:29]: updated npm packages

File: wallet-blockchain-interface/src/chains/stacks/helper-functions.ts (hunk 2 above)
Comment (scolear) [reply wallet-blockchain-interface/src/chains/stacks/helper-functions.ts:56]: this was used for the Redstone stuff, which we currently don't do anymore. gonna remove

**Pull request merged**